src/server/origin/address/controller.js

```diff
@@ -50,6 +50,7 @@ export const originAddressPostController = {
     }))
 
     if (!isValid) {
+      req.yar.clear('originAddress')
```

Comment:
`originAddress` is no longer the session field - it's `origin.address` isn'tit?

Reply:
I'm not 100% sure we should clear state on failure, FWIW. 

Imagine the scenario where: 

1. user is on the summary page
2. user clicks `change` against address
3. user attempts a change and gets an error back
4. user navigates back to summary page

I'm not sure if it's most intuitive that the address is wiped out, or if it's what it was before a failed attempt.

Reply:
i would argue thats less intuitive, theres a chance user might think they have changed the address or at least said "do not go to the address previously entered" anymore, but if we keep the previously entered one then we have no way of checking the address is in fact what is intended and mark as valid

Reply:
But also, if we're going down this path then I suspect we should add it to all cases?

Reply:
have just done so in the commits below too (i noticed the same thing)